src/Regis/Application/Model/Git/Diff/File.php

```diff
@@ -66,4 +67,23 @@ public function getChanges(): array
     {
         return $this->changes;
     }
+
+    public function findPositionForLine(int $line): int
+    {
+        $changes = $this->getChanges();
+
+        /** @var Change $change */
+        foreach ($changes as $change) {
```

Comment:
Avoid using static access to class '\Regis\Application\Inspection\Exception\LineNotInDiff' in method 'findPositionForLine'.